gotools.mk

```diff
@@ -0,0 +1,14 @@
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+GOTOOLS = $(shell grep '_' tools/tools.go | cut -d\" -f2 | rev | cut -d/ -f1 | rev)
+
+.PHONY: gotools
+gotools: $(patsubst %,.build/tools/%, $(GOTOOLS))
+
+.build/tools/%: tools/go.mod tools/tools.go
+	@mkdir -p $(@D)
+	@$(eval TOOL = ${subst .build/tools/,,${@}})
+	@$(eval FQP = $(shell grep "_" tools/tools.go | cut -d "_" -f2 | grep ${TOOL}\"$))
```

Comment:
Do you have an issue with grepping the URL out of the tools.go file? Otherwise I need to maintain a map like we have in the Fabric repo of the URL's, and that's really what I was trying to avoid with grepping the tools out of the tools.go file above